samples/QR-app-installation/csharp/README.md

```diff
@@ -0,0 +1,96 @@
+﻿This sample demos app installation using QR code with team id in Csharp
+
+This feature shown in this sample is in Public Developer Preview and is supported in desktop and mobile.
```

Comment:
Here, replace it with "Currently, Microsoft Teams support for QR or barcode scanner capability is only available for mobile clients."